main.go

```diff
@@ -134,8 +134,7 @@ func main() {
 		if conf.Auth.Enabled {
 			fmt.Printf("Demo mode is not supported with auth enabled")
```

Comment:
is this still relevant?

Reply:
Looks like you are looking at older commit.  have a look at latest commit (Better go for Files Changed tab in the merge request